### DISCUSSION TITLE
绳网的节奏向性内容

### DISCUSSION THREAD

**Discussion by JiuXiat** [share121/inter-knot#5156] [其他]
![1725238918591](https://github.com/user-attachments/assets/e78db079-d557-4ebd-9f35-7823c952d268)

**何为节奏向性内容？**
这指的是煽动负面情绪，具有引战性质的类容，才叫节奏向性内容
这种帖子是要在绳网明令禁止的
巡海游侠们要见一个封一个
**还是有什么不懂的？**
现在绳网更新了一个搜索功能，你去搜索：绳网的规则与准则，那里面有大大小小的关于绳网禁止做的事
节奏内容列举了一大部分
好了，祝各位发帖人和游客以及巡海游侠
中秋安康！

**Comment by Fluorine-z** [comment]
为了寻猎！
